### ISSUE TITLE
`spin` broken in Safari

### ISSUE THREAD

**Issue by versoworks** [Templarian/MaterialDesign-React#68]
Using Safari v15.4, `spin` property does not work. Tested on https://templarian.github.io/@mdi/react/

@Templarian have you had an opportunity to investigate this issue?

**Comment by Haosik** [comment]
Might help someone, as a workaround for my "spinner loader" with `mdiLoading` (and styled-component) I ended up with 
```
const RotatingIcon = styled(Icon)`
  animation: ${rotate} 1s linear infinite;
`
```
and then rendering it like 
```
<RotatingIcon
      path={mdiLoading}
      horizontal
      vertical
     ... some other props here but without spin
    />
```
It makes rotating the whole `svg` which appears to work in Safari.

**Comment by ngburke** [comment]
@Templarian if someone puts up a pull request for this, would you consider merging it? Thanks!

**Comment by Templarian** [comment]
@ngburke Yes. This fell off my radar with the busy summer. 100% would merge a PR for this.

**Comment by ngburke** [comment]
@Templarian PR  #72 up for your review, thanks.

**Comment by Templarian** [comment]
Released `1.6.1`
